package.json

```diff
@@ -10,41 +10,37 @@
     "check-fmt": "prettier --check .",
     "format": "prettier --write .",
     "lint": "eslint src/**/*.ts",
-    "prebuild": "cp package.json ./src/package.json",
+    "prebuild": "echo 'export const pkg =' > src/package.ts; cat package.json >> ./src/package.ts && prettier --write ./src/package.ts && eslint --fix ./src/package.ts",
```

Comment:
Could you clarify why this is here? Feels quite icky, and just want to make sure it's truly necessary.

Reply:
I want to embed the version from the package.json in the metrics being submitted without maintaining it in two places. This seems to work fine as package.json when depended upon in other actions, but when I run run-test.js after minification, it doesn't work with complaints that I need need to import like this: `import info from `./package.json` assert { type: 'json' };` or like this: `import info from `./package.json` with { type: 'json' };`, and that causes other issues.

Reply:
Made much nicer in https://github.com/DeterminateSystems/detsys-ts/pull/16/commits/c65adbc917507d589245819a34b3983236aa5027